Rubberduck.Core/UnitTesting/IFakes.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Rubberduck.UnitTesting
+{
+    public interface IFakes
+    {
+        void StartTest();
+        void StopTest();
+    }
+
+    public interface IFakesFactory
+    {
+        IFakes GetFakes();
+    }
```

Comment:
I presume this is an auto-magic factory handled by CW. If so, then this is missing the `Release` method.

Reply:
umm ... this didn't have a release method before either. Are you sure we need it?

Reply:
This factory interface has a concrete implementation that gets explicitly registered in `RubberduckIoCInstaller`. So, no `Release` method is required.

Reply:
@MDoerner that is not correct. I removed the concrete implementation when rewriting how the IFakesProvider is injected...